auth_session_timeout/__openerp__.py

```diff
@@ -9,7 +9,8 @@
     'summary': """
         This module disable all inactive sessions since a given delay""",
 
-    'author': "ACSONE SA/NV, Dhinesh D, Odoo Community Association (OCA)",
+    'author': "ACSONE SA/NV, Dhinesh D, Jesse Morgan, \
+Odoo Community Association (OCA)",
```

Comment:
If you don't like escaped new lines, you can also use multiple strings, like this:
```
    'author': 'ACSONE SA/NV, Dhinesh D, Jesse Morgan, '
              'Odoo Community Association (OCA)',
```